modules/ROOT/pages/alerts.adoc

```diff
@@ -23,16 +23,27 @@ Note that thresholds are checked every five minutes. The evaluated query is the
 * Operational alerts for APIs through API Manager
 * Operational alerts for server and app events through Runtime Manager
 
-== Use the Alerts Page
+== Access the Alerts Page
 
-After you set up alerts, you can enable or disable any of them, filter alerts based on the <<alert_status, alert status>>, filter alerts based on the <<alert_severity, alert severity>>, and link to the graph in a custom dashboard that is associated with the alert.
+include::partial$include-nav-steps-no-permissions.adoc[]
 
-The *Alerts* page shows basic alerts for servers and apps or advanced alerts (https://www.mulesoft.com/anypoint-pricing[Titanium subscription required]) for graphs in custom dashboards.
+. In the Anypoint Monitoring navigation menu, click *Alerts*.
+
+In the *Alerts* page, you can:
+
+* Create basic alerts
+* View:
```

Comment:
Slightly easier to read as just one level? You don't do create 2nd level for 'Filter by.' Since there are just two "View" items, change to one level (repeat View)?

Reply:
Added. Thank you.